### PR TITLE
fix: add 1px spacing around settings icon

### PR DESCRIPTION
**Pull request by mittalyashu**
| Before | After |
| --- | --- |
| CleanShot 2021-02-05 at 10 36 05@2x | CleanShot 2021-02-05 at 10 33 57@2x |

### CODE REVIEW

**Comment by ericfennis** [comment]
@mittalyashu Maybe the circle in the middle a bit larger?

File: icons/settings.svg [hunk 1]
```diff
@@ -9,6 +9,6 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <circle cx="12" cy="12" r="3" />
-  <path d="M19.4 15a1.65 1.65 0 00.33 1.82l.06.06a2 2 0 010 2.83 2 2 0 01-2.83 0l-.06-.06a1.65 1.65 0 00-1.82-.33 1.65 1.65 0 00-1 1.51V21a2 2 0 01-2 2 2 2 0 01-2-2v-.09A1.65 1.65 0 009 19.4a1.65 1.65 0 00-1.82.33l-.06.06a2 2 0 01-2.83 0 2 2 0 010-2.83l.06-.06a1.65 1.65 0 00.33-1.82 1.65 1.65 0 00-1.51-1H3a2 2 0 01-2-2 2 2 0 012-2h.09A1.65 1.65 0 004.6 9a1.65 1.65 0 00-.33-1.82l-.06-.06a2 2 0 010-2.83 2 2 0 012.83 0l.06.06a1.65 1.65 0 001.82.33H9a1.65 1.65 0 001-1.51V3a2 2 0 012-2 2 2 0 012 2v.09a1.65 1.65 0 001 1.51 1.65 1.65 0 001.82-.33l.06-.06a2 2 0 012.83 0 2 2 0 010 2.83l-.06.06a1.65 1.65 0 00-.33 1.82V9a1.65 1.65 0 001.51 1H21a2 2 0 012 2 2 2 0 01-2 2h-.09a1.65 1.65 0 00-1.51 1z" />
+  <path d="M12 14a2 2 0 100-4 2 2 0 000 4z" />
```
Comment (ericfennis) [inline icons/settings.svg:12]:
```suggestion
  <circle cx="12" cy="12" r="3" />
```

**Comment by mittalyashu** [comment]
@ericfennis Wouldn't that collide the inner circle on increasing the `stroke-width`

**Comment by ericfennis** [comment]
@mittalyashu It's is matching our icon design guidelines.
image

**Comment by mittalyashu** [comment]
In that case, this PR LGTM.

**Review by locness3 — APPROVED** [review]

**Pull request merged**